src/notifications/generate.ts

```diff
@@ -143,6 +146,15 @@ export const generateNotificationMap: Record<
       .avatarSource(ctx.source)
       .uniqueKey(ctx.bookmark.remindAt.toString())
       .objectPost(ctx.post, ctx.source, ctx.sharedPost),
+  streak_reset_restore: (builder, ctx: NotificationStreakContext) =>
+    builder
+      .icon(NotificationIcon.Streak)
+      .description('Click here if you wish to restore your streak')
+      .uniqueKey(ctx.streak.lastViewAt.toString())
```

Comment:
could we make it per day (without time portion) eg `19-08-2024` so we don't duplicate notifications for any reason in the same day?

Reply:
Yeah, was thinking of that too at first, just realized it is unlikely to happen since reset can only trigger after some days, but I agree, we can do that to ensure duplicates will be avoided.